building/configlet/lint.md

```diff
@@ -330,3 +353,17 @@ The `config.json` file should have the following checks:
    > - Lowercase prepositions, regardless of length, except when they are stressed, are used adverbially or adjectivally, or are used as conjunctions.
    > - Lowercase the words _to_ and _as_.
    > - Lowercase the second part of Latin species names.
+4. Valid `files` pattern: A non-blank string¹ that specifies a location of a file used in an exercise, relative to the exercise's directory. A pattern may use one of the following placeholders:
+- `%{kebab_slug}`: the `kebab-case` exercise slug (e.g. `bit-manipulation`)
+- `%{snake_slug}`: the `snake_case` exercise slug (e.g. `bit_manipulation`)
+- `%{camel_slug}`: the `camelCase` exercise slug (e.g. `bitManipulation`)
+- `%{pascal_slug}`: the `PascalCase` exercise slug (e.g. `BitManipulation`)
+5. Unique version 4 UUID string: A string that satisfies all of these conditions:
+   - It only exists once in the track-level `config.json` file of a given Exercism track
+   - It does not exist in the track-level `config.json` file of any other Exercism track
+   - It does not exist in any `canonical-data.json` file in https://github.com/exercism/problem-specifications
+   - It matches the regular expression:
+   ```
+   ^[0-9a-f]{8}-[0-9a-f]{4}-4[0-9a-f]{3}-[89ab][0-9a-f]{3}-[0-9a-f]{12}$
+   ```
+   You can run `configlet uuid` to generate a suitable UUID.
```

Comment:
Could we add an example along with the regexp pls?

Reply:
Good idea. I pushed https://github.com/exercism/docs/pull/118/commits/8145c40cd6e42be625aeb855ef9928bc74dd9120 - is that what you had in mind?

Reply:
Perfect. Thanks :)

Let's see how many times people just copy+paste this now 🤦 😉